netlify/functions/presence.js

```
@@ -4,80 +4,120 @@ const sql = neon();
           statusCode: 200,
           headers,
           body: JSON.stringify(activeUsers),
         };
 
-      case 'POST':
-        // Update user presence (heartbeat)
-        const { username } = JSON.parse(body);
-        
+      case "POST":
+        // Update user presence (heartbeat) and typing status
+        const { username, isTyping } = JSON.parse(body);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)